spec/acceptance/nodesets/archlinux-2-x64.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Should we exclude this (and others) to keep the noise down?

Reply:
Sorry.  This has come up before.  Fixed.